flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/FileType.java

```diff
@@ -22,6 +22,18 @@
  * Different file types to request from the {@link TaskExecutor}.
  */
 public enum FileType {
+	/**
+	 * log
```

Comment:
Can you add some more expressive JavaDocs here?

Reply:
done.